Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -499,14 +503,27 @@ class SmartPage_Context
   public SetCacheDetectionFieldLoaded(): void
   {
     window.sessionStorage.setItem(this.GetPageToken(), SmartPage_CacheDetection[SmartPage_CacheDetection.HasLoaded]);
+    this.ClearOldPageTokenSessionStorageEntries();
   };
 
   // Marks the page as submitted.
   public SetCacheDetectionFieldSubmitted(): void
   {
     window.sessionStorage.setItem(this.GetPageToken(), SmartPage_CacheDetection[SmartPage_CacheDetection.HasSubmitted]);
+    this.ClearOldPageTokenSessionStorageEntries();
   };
 
+  private ClearOldPageTokenSessionStorageEntries(): void
+  {
+    const pageTokenEntries = Object.keys(window.sessionStorage).map((k: string) => +k).filter(t => !isNaN(t));
```

Comment:
what's the `+k` syntax here?

And I think we should add a prefix, not just use a number, now that we a) limit the count and b) perform active cleanup. `pt_` or `PageToken_`

also, for an extra performance boost: we could check if the total number exceeds the keepcount before checking if that number is acutally pagetokens. until such a time as the page uses many own values in the session store, the first check will be very efficient.

As an alternative, we could (should?) also persist the actual count of pagetokens in the sessionstore. that way, we can do a quick lookup before parsing all the keys. Should there ever be another heavy user, this check would prove beneficial to the performance impact.